ls1intum/Artemis
# `Development`: Fix logging statement for online course client registrations

### Checklist
#### General


#### Server

### Motivation and Context


For online courses that aren't linked to an external system or use the older LTI v1.0 standard, some onlineCourseConfiguration fields are left blank. This results in an issue when attempting to construct the ClientRegistration for such courses. Currently, this issue is flagged as an error.

![image](https://github.com/ls1intum/Artemis/assets/11020241/3605b6be-4e79-419d-9ff0-aa445aaf8898)


### Description

- Adjusted the logging mechanism to categorize these as warnings instead of errors.
- Enhanced the warning message to provide more details.

### Steps for Testing

Please test locally to see the adjusted log message.

Prerequisites:
- 1 Course with Online Course enabled

1. Start Artemis server
2. Verify log warning for newly defined Online Couse 

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage






no change

### Screenshots


![image](https://github.com/ls1intum/Artemis/assets/11020241/874a0e3d-c074-470e-9c05-76bdfec25233)
